packages/@aws-cdk/aws-apigateway/lib/integrations/mock.ts

```diff
@@ -0,0 +1,22 @@
+import { Integration, IntegrationOptions, IntegrationType } from '../integration';
+
+/**
+ * This type of integration lets API Gateway return a response without sending
+ * the request further to the backend. This is useful for API testing because it
```

Comment:
I don't find it super useful for testing, it's useful when you're developing your API & client at the same time and you want the client to be able to enact some not-yet-implemented APIs... But that's it.

It *is* definitely useful when your response is static or mostly static, as it's a cheap and incredibly fast way to get it out of the door :)

Reply:
I copied the documentation from the AWS docs, so I rather just leave it like that.